pkg/pillar/agentlog/agentlog.go

```diff
@@ -199,11 +201,37 @@ func handleSignals(log *base.LogObject, agentName string, agentPid int, sigs cha
 				if sigUsr2File != nil {
 					sigUsr2File.Close()
 				}
+				go listenDebug()
 			}
 		}
 	}
 }
 
+func listenDebug() {
+	mux := http.NewServeMux()
+
+	server := &http.Server{
```

Comment:
Did you see the golangcilint warning in https://github.com/lf-edge/eve/pull/3229/files?